TTC.Deployment.Tests/AssumeRoleTest.cs

```diff
@@ -19,27 +15,20 @@ public class AssumeRoleTest
     {
         AwsConfiguration _awsConfiguration;
         AmazonIdentityManagementServiceClient _iamClient;
-        AmazonSecurityTokenServiceClient _securityTokenServiceClient;
         AmazonS3Client _s3Client;
-        User _user;
-        BasicAWSCredentials _userCredentials;
-        
-        Role _role;
-        string _roleName;
-
+        
         private readonly string _userName = "aws_tools_assume_role_test_user";
-        private readonly string _bucketName = "aws-tools-test-bucket-1";
+        private readonly string _bucketName = "aws-tools-test-bucket-00";
```

Comment:
If this is global, what happens if many ppl run this tool .. could  there be a collision?  Perhaps we need to append a datestamp when running ...